common/views/components/PageHeader/PageHeader.js

```diff
@@ -126,6 +125,25 @@ export function getHeroPicture(
   );
 }
 
+function addFreeLabel(labelsList) {
+  const freeLabel = {
+    text: 'Free',
+    labelColor: 'black',
+  };
+  if (labelsList) {
+    return {
+      ...labelsList,
+      ...{
+        labels: [freeLabel, ...labelsList.labels],
+      },
+    };
+  } else {
+    return {
+      labels: [freeLabel],
+    };
+  }
+}
+
 type Props = {|
   breadcrumbs: ElementProps<typeof Breadcrumb>,
   labels: ?ElementProps<typeof LabelsList>,
```

Comment:
bikeshed: this should really be called `labelsListProps`, it's quite confusing otherwise.